azure_functions_worker/constants.py

```diff
@@ -56,4 +56,4 @@
 # Paths
 CUSTOMER_PACKAGES_PATH = os.path.join(pathlib.Path.home(),
                                       pathlib.Path(
-                                          "site/wwwroot/.python_packages"))
+    "site/wwwroot/.python_packages/lib/site-packages"))
```

Comment:
The indentation - is flake working?

Reply:
Also - adding /lib/site-packages - would it work? we should also explore if we need another check if site/wwwroot/.python_packages/ exists

And why not add /home here as well? to simplify it

Reply:
For python 3.7+ customer packages should be at /home/site/wwwroot/.python_packages/lib/site-packages, so this should fix the issue